crates/services/block_aggregator_api/proto/api.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package blockaggregator;
```

Comment:
Is there a way to generate protobuf schema from the rust type?

Reply:
https://github.com/tokio-rs/prost/issues/1350

Let's see what they will suggest